lib/process-services-cloud/src/lib/process/process-list/components/process-list-cloud.component.html

```diff
@@ -3,6 +3,7 @@
         [columns]="columns"
         [stickyHeader]="stickyHeader"
         [loading]="isLoading"
+        [sorting]="sortInput"
```

Comment:
see below, rename to `sorting` to be intuitive
```suggestion
        [sorting]="sorting"
```